src/qemu.rs

```diff
@@ -626,6 +626,7 @@ impl Qemu {
         let mut c = Command::new(format!("qemu-system-{}", target.arch));
 
         c.args(QEMU_DEFAULT_ARGS)
+            .stderr(Stdio::piped())
```

Comment:
For interactive mode, I wasn't sure if qemu sends stderr generated in guest out to stderr or stdout. But from this test, looks like it all goes to stdout:

```
$ vmtest -k ~/dev/linux/arch/x86/boot/bzImage - 2>/tmp/stderr
[...]
[root@archlinux /]# echo stdout
stdout
[root@archlinux /]# echo stderr >&2
stderr
[root@archlinux /]#
logout
Failed to write utmp record: Permission denied
Powering off.
[   85.886972] ACPI: PM: Preparing to enter system sleep state S5
[   85.888184] kvm: exiting hardware virtualization
[   85.888560] reboot: Power down
$ wc -l /tmp/stderr
0 /tmp/stderr
```

So I think this is not only correct, but an improvement. As errors will be correctly parsed by `vmtest` and not just sent to terminal for interactive mode.

Reply:
Good catch, LGTM.